### DISCUSSION TITLE
Possible to use pipeline in dynamic mode?

### DISCUSSION THREAD

**Discussion by moon-watch** [aunetx/blur-my-shell#579] [Ideas]
![截图 2024-04-21 16-37-12](https://github.com/aunetx/blur-my-shell/assets/115158883/9da6dd86-5a8a-4a0c-9b57-b1cf4e16d389)
Dynamic mode with dynamic background.
![截图 2024-04-21 16-37-17](https://github.com/aunetx/blur-my-shell/assets/115158883/e1c49b7f-275c-471e-86b4-867e70a55288)
Static mode with only the wallpaper as the fixed background.
It looks better when use dynamic blur on dash to dock, but i can not find any option to set the corners' radius or use the pipeline to set it.

**Comment by aunetx** [comment]
Hello, this is not possible for the moment sorry (and may never be), because of how the dynamic blur effect works and that I can't control. I am however kinda working on this to try to see what I could do, but at this points it's better to forget it until it is correctly implemented upstream!

**Comment by moon-watch** [comment]
Thanks for your rapid reply!
Maybe you mean that it should be implemented by dash to dock?

**Comment by aunetx** [comment]
Unfortunately, even dash-to-dock can't do anything about it, it is really up to the GNOME Shell team to choose implementing it! Although I could probably hack together something that works, but that would be a lot of work and is out of my league really
